packages/tenant-process/src/services/tenantService.ts

```diff
@@ -699,4 +719,54 @@ function reassignDeclaredAttribute(
   );
 }
 
+function assignVerifiedAttribute(
+  targetTenant: Tenant,
```

Comment:
```suggestion
  attributes: TenantAttribute[],
```
Same, you only need the attributes, not the whole Tenant